bots/naughty/fedora-27/9282-docker-cgroup-failure

```diff
@@ -0,0 +1,7 @@
+Traceback (most recent call last):
```

Comment:
According to the downstream bug, this is not an issue on Fedora 27.  Perhaps it makes sense to update that bug with information that F27 is also broken?  Or is this a different issue?

Reply:
No, same issue. I updated the downstream bug.